### PR TITLE
fixes issue 779

### PR DESCRIPTION
**Pull request by arp95**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -11,10 +11,10 @@ def SUPPORT_LIB_VERSION = "24.2.0"
 
 android {
     compileSdkVersion 24
-    buildToolsVersion "24.0.2"
+    buildToolsVersion "24.0.0"
 
     dexOptions {
-        // Skip pre-dexing when running on Travis CI or when disabled via -Dpre-dex=false.
+         Skip pre-dexing when running on Travis CI or when disabled via -Dpre-dex=false.
```
Comment (the-dagger) [inline android/app/build.gradle]: Please don't modify the gradle files

**Comment by arp95** [comment]
@the-dagger i made the build.gradle the same as before.
the ui changes i made were in track display where a list of tracks are displayed. there i made changes as you can see over here.
14923116_10209504516284271_2090077525_o

**Comment by arp95** [comment]
final change to the track items. Please review and merge if possible @the-dagger so that i can work on the other issues.

File: android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java [hunk 2]
```diff
@@ -91,7 +91,7 @@ public void onBindViewHolder(SessionViewHolder holder, int position) {
                 + ISO8601Date.getTimeZoneDateString(ISO8601Date.getDateObject(session.getStartTime())).split(",")[1];
 
         holder.sessionTitle.setText(session.getTitle());
-        holder.sessionAbstract.setText(session.getSummary());
+        //holder.sessionAbstract.setText(session.getSummary());
```
Comment (the-dagger) [inline android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java]: @arp95 Why did you remove this?
We need the abstract for each session in the card.
